cli: html stories homogenization

tooppaaa opened the pull request:
Issue:

We need similar stories after generation to run e2e against

Part of #10702

tooppaaa committed:
cli: html stories homogenization

yannbf commented on lib/cli/src/generators/HTML/template-csf/stories/0-Welcome.stories.js:
```diff
@@ -0,0 +1,48 @@
+    Just like that, you can add your own components as stories. <br />
+    You can also edit those components and see changes right away. <br />
+    (Try editing the <span class="inline-code">Button</span> stories located at
+    <span class="inline-code">src/stories/index.js</span>.)
```
the path should refer to `1-Button.stories.js `

gaetanmaisse replied:
Done

yannbf commented on lib/cli/src/generators/HTML/index.js:
```diff
@@ -9,7 +9,13 @@ import {
+    '@storybook/html',
+    '@storybook/addon-actions',
+    '@storybook/addon-links',
+    '@storybook/addons',
```
can we check if `@storybook/addons` is necessary?

gaetanmaisse replied:
Not needed, see https://github.com/storybookjs/storybook/pull/10703#issuecomment-628227892

yannbf reviewed (changes requested):
LGTM! only a simple thing to change

gaetanmaisse committed:
feat(cli): remove `@storybook/addons` from deps installed when running `sb init` in an html project

`@storybook/addons` is already a dependency of all `@storybook/addon-XXX` packages so there is no need to add it to the root project directly.

gaetanmaisse committed:
fix(cli): fix reference to Button story in story template used when init an SB in an html project

gaetanmaisse committed:
Merge branch 'next' into generator_html

yannbf reviewed (approved):
Awesome stuff!
